packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1411,20 +1434,19 @@ export class ProjectService extends BaseService {
                     projectUuid,
                 },
             });
+            const projectWithWareouse = {
+                ...updatedProject,
+                warehouseConnection: updatedProject.warehouseConnection,
```

Comment:
it feels redundant, but updatedProject has an optional warehouseConnection, however, we have a check for undefined a few lines above. So forces it to be mandatory, like in CreateProject